account_manager/src/validator/recover.rs

```diff
@@ -148,7 +148,7 @@ pub fn cli_run(matches: &ArgMatches, validator_dir: PathBuf) -> Result<(), Strin
         println!(
             "{}/{}\tIndex: {}\t0x{}",
             index - first_index,
```

Comment:
This is option 2:
If count = 2, the UX would be:
```
1/2
2/2 
```
This would be more intuitive from the user perspective. I would opt for this one too
```suggestion
            index - first_index + 1,
```